applications/luci-app-spoofer/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2018 The Regents of the University of California
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=Spoofer LuCI interface
+LUCI_DEPENDS:=+luci-mod-admin-full +spoofer
+
+PKG_MAINTAINER:=Ken Keys <kkeys@caida.org>
+PKG_LICENSE:=GPL-3.0
```

Comment:
Is this correct for the luci app?  What's the intention on how that interacts with other luci apps? Note that this is independent of the license for the spoofer application itself.

Reply:
The only intention was to license this as open source; I wasn't aware this had any effect on interaction with other luci apps.  Is there an explanation somewhere?  What would you suggest?  I'm sure any reasonable open source license would be acceptable on my end.

Reply:
@kenkeys - according to http://www.apache.org/licenses/GPL-compatibility.html there seems to be an issue with mixing GPL v3.0 software with Apache 2.0 (which most of LuCI is licensed as).

Maybe simply relicense to Apache 2.0, then there shouldn't be a problem.